update_copyrights

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+# encoding: utf-8
+
+##
+# This script checks all media files in the repository for whether they've been modified or added without updating the file tracking image, sound, and music copyright
+##
+
+import argparse
+import contextlib
+import csv
+from operator import itemgetter
+import os
+from pathlib import Path
+from subprocess import check_output
+import sys
+
+def do_git(file):
+    # print("started : " + file)
+    result = str(check_output(["git", "log", "-1", "--format=%ad,%an,,,,", "--date=format:%Y/%m/%d", file]), 'UTF-8').split(sep=",")
+    result[1] = result[1][2:]
```

Comment:
I think this should assign `file[2:]` now. though field 2 from the git output is the author name...

Also not sure if the `[2:]` is correct on windows. I assume it's supposed to remove `./`. Looks like `os.path.normpath` might be better.